pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1184,6 +1185,8 @@ type httpListenerOpts struct {
 	// should be added.
 	addGRPCWebFilter bool
 	useRemoteAddress bool
+
+	supportHTTP3 bool
```

Comment:
I think this needs a comment/rename. "Support" seems to imply it _additionally_ supports HTTP3 - not that it is _only_ HTTP3

Reply:
I changed it to `http3Only`.